examples/app-with-wasm/webpack.config.ts

```diff
@@ -0,0 +1,19 @@
+import type { Configuration } from 'webpack';
+import { merge } from 'webpack-merge';
+import grafanaConfig from './.config/webpack/webpack.config';
+import pluginJson from './src/plugin.json';
+
+const config = async (env): Promise<Configuration> => {
+  const baseConfig = await grafanaConfig(env);
+  return merge(baseConfig, {
+    // Add custom config here...
+    experiments: {
+      asyncWebAssembly: true,
+    },
+    output: {
+      publicPath: `public/plugins/${pluginJson.id}/`,
+    },
+  });
+};
+
```

Comment:
I know this PR is quite old but it's worth mentioning that this isn't necessary any more with latest create-plugin configs it should work out the box.

Reply:
That's true! In fact I think enough things have been upstreamed into create-plugin that this PR is no longer really needed; everything should just work without any custom webpack config now :pray: